.github/workflows/ci.yml

```diff
@@ -55,6 +55,6 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-          CCM_CLUSTER_START_DEFAULT_TIMEOUT: 600
+          CCM_CLUSTER_START_TIMEOUT_OVERRIDE: 600
```

Comment:
Slightly curious: why wasn't the `CCM_CLUSTER_START_DEFAULT_TIMEOUT` working?

Reply:
I think the github CI environment is quite constrained and things are slow to startup?

Reply:
Because the DSE call path in `ccm` has a hard-coded timeout: https://github.com/riptano/ccm/blob/master/ccmlib/dse_cluster.py#L78

I did not want to change default behaviour in `ccm`